tests/countries/test_norway.py

```diff
@@ -21,6 +21,13 @@ class TestNorway(CommonCountryTests, SundayHolidays, TestCase):
     def setUpClass(cls):
         super().setUpClass(Norway)
```

Comment:
You need to `import warnings` at the very top as well, see French test case example here:
```python
import warnings
from unittest import TestCase

from holidays.countries.france import France, FR, FRA
from tests.common import CommonCountryTests
```